src/IncludedCommands/UsefulFun/fly.lua

```diff
@@ -247,42 +267,101 @@ return {
                         (Flight :: any):Destroy()
                     end))
                     table.insert(Flight.Events, UserInputService.InputBegan:Connect(function(Key, Processeed)
-                        if Processeed and Key.KeyCode ~= Enum.KeyCode.LeftShift and Key.KeyCode ~= Enum.KeyCode.RightShift then return end
+                        if Processeed and Flight.MultiPlatform[Key.KeyCode] ~= Enum.KeyCode.LeftShift and Flight.MultiPlatform[Key.KeyCode] ~= Enum.KeyCode.RightShift then return end
     
                         --Handle the key.
-                        if Key.KeyCode == Enum.KeyCode.E then
-                            --Toggle the flight.
-                            if Flight.Active then
-                                (Flight :: any):Stop()
-                            else
-                                (Flight :: any):Start()
-                            end
-                        elseif Flight.KeysDown[Key.KeyCode] ~= nil then
+                        if Flight.MultiPlatform[Key.KeyCode] ~= nil then
                             --Update the key press.
-                            Flight.KeysDown[Key.KeyCode] = true
+                            Flight.KeysDown[Flight.MultiPlatform[Key.KeyCode]] = true
                             Flight:UpdateMultipliers()
                         end
                     end))
+                    table.insert(Flight.Events, UserInputService.InputChanged:Connect(function(Key)
+                        local X,Y = nil,nil
+                        if Key.KeyCode == Enum.KeyCode.Thumbstick1 then
+                            X = Key.Position.X
+                            Y = Key.Position.Y
+                        elseif Key.UserInputType == Enum.UserInputType.Touch then
+                            local camera = workspace.CurrentCamera
+                            local movementDirection = camera.CFrame:vectorToObjectSpace(Flight.Humanoid.MoveDirection).Unit
+                            X = movementDirection.X
+                            Y = movementDirection.Z * -1
+                        end
+
+                        if X and Y then
+                            if X <= -.5 then
+                                Flight.KeysDown[Enum.KeyCode.A] = true
+                                Flight.KeysDown[Enum.KeyCode.D] = false
+                            elseif X >= .5 then
+                                Flight.KeysDown[Enum.KeyCode.A] = false
+                                Flight.KeysDown[Enum.KeyCode.D] = true
+                            else
+                                Flight.KeysDown[Enum.KeyCode.A] = false
+                                Flight.KeysDown[Enum.KeyCode.D] = false
+                            end
+
+                            if Y <= -.5 then
+                                Flight.KeysDown[Enum.KeyCode.S] = true
+                                Flight.KeysDown[Enum.KeyCode.W] = false
+                            elseif Y >= .5 then
```

Comment:
All other decimals in the project include the leading zero. Please include it for consistency.
```suggestion
                            if X <= -0.5 then
                                Flight.KeysDown[Enum.KeyCode.A] = true
                                Flight.KeysDown[Enum.KeyCode.D] = false
                            elseif X >= 0.5 then
                                Flight.KeysDown[Enum.KeyCode.A] = false
                                Flight.KeysDown[Enum.KeyCode.D] = true
                            else
                                Flight.KeysDown[Enum.KeyCode.A] = false
                                Flight.KeysDown[Enum.KeyCode.D] = false
                            end

                            if Y <= -0.5 then
                                Flight.KeysDown[Enum.KeyCode.S] = true
                                Flight.KeysDown[Enum.KeyCode.W] = false
                            elseif Y >= 0.5 then
```